.travis.yml

```diff
@@ -7,6 +7,9 @@ services:
 install:
   - pip install pipenv
   - pipenv install
+  - npm install
+  - npm install babel-core@^7.0.0-bridge.0 --save-dev --dev jest --save-dev vue-jest vue-template-compiler --save-dev jest-serializer-vue
```

Comment:
Installing specific packages shouldn't be necessary.  If we install them with the --save in development, the package.json should reflect this dependency and npm install should be all we need.